sdk/appservice/arm-appservice/LICENSE.txt

```diff
@@ -1,21 +1,21 @@
-The MIT License (MIT)
-
-Copyright (c) 2018 Microsoft
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+The MIT License (MIT)
```

Comment:
@kpajdzik do you know why each management package has its own copy of the license file? There shouldn't be any reason we cannot share one for the entire repo.

Reply:
I don't think NPM is going to display correct license if it isn't shipped with the package.

Reply:
I don't even know if the file is necessary for the standard licenses like MIT as the package.json simply has ` "license": "MIT",` but the file does automatically get included in the package according to the spec https://docs.npmjs.com/files/package.json#files so if we did move it we would have to add an entry for it in our package.json manaully. For now lets leave it but it is something we should potentially clean-up at some point.

Reply:
Hmm, you may be right.

Reply:
I filed issue https://github.com/Azure/azure-sdk-for-js/issues/2150 to follow-up on this conversation.